README.rst

```diff
@@ -121,40 +121,40 @@ Getting Started
 
 
 Here is an example on how to use the ``GLM`` estimator.
+
 .. This example is also found in examples/intro_example.py.
 
 .. code:: python
    
-   import numpy as np
-   import scipy.sparse as sps
-   from pyglmnet import GLM, simulate_glm
-
-   # create an instance of the GLM class
-   glm = GLM(distr="poisson")
-
-   # sample random coefficients
-   n_samples, n_features = 1000, 100
-   beta0 = np.random.normal(0.0, 1.0, 1)
-   beta = sps.rand(n_features, 1, 0.1)
-   beta = np.array(beta.todense())
-
-   # simulate training data
-   X_train = np.random.normal(0.0, 1.0, [n_samples, n_features])
-   y_train = simulate_glm("poisson", beta0, beta, X_train)
-
-   # simulate testing data
-   X_test = np.random.normal(0.0, 1.0, [n_samples, n_features])
-   y_test = simulate_glm("poisson", beta0, beta, X_test)
-
-   # fit the model on the training data
-   #scaler = StandardScaler().fit(X_train)
-   glm.fit(X_train, y_train)
-
-   # predict using fitted model on the test data
-   yhat_test = glm.predict(X_test)
-
-   # score the model
-   deviance = glm.score(X_test, y_test)
+    import numpy as np
+    import scipy.sparse as sps
+    from pyglmnet import GLM, simulate_glm
+    
+    # create an instance of the GLM class
+    glm = GLM(distr="poisson")
+    
+    # sample random coefficients
+    n_samples, n_features = 1000, 100
+    beta0 = np.random.normal(0.0, 1.0, 1)
+    beta = sps.rand(n_features, 1, 0.1)
+    beta = np.array(beta.todense())
+    
+    # simulate training data
+    X_train = np.random.normal(0.0, 1.0, [n_samples, n_features])
+    y_train = simulate_glm("poisson", beta0, beta, X_train)
+    
+    # simulate testing data
+    X_test = np.random.normal(0.0, 1.0, [n_samples, n_features])
+    y_test = simulate_glm("poisson", beta0, beta, X_test)
+    
+    # fit the model on the training data
+    glm.fit(X_train, y_train.squeeze())
```

Comment:
could you index it instead of using squeeze. As in y_train[0] or y_train[:, 0] whatever may be the case